pkgs/by-name/si/simplex-chat-desktop/package.nix

```diff
@@ -1,15 +1,19 @@
 { lib
 , appimageTools
 , fetchurl
+, nix-update-script
 }:
 
 let
   pname = "simplex-chat-desktop";
-  version = "5.4.2";
+  version = "5.4.3";
 
   src = fetchurl {
     url = "https://github.com/simplex-chat/simplex-chat/releases/download/v${version}/simplex-desktop-x86_64.AppImage";
-    hash = "sha256-t9wFOKGmy/mGFtETv1EkturAM4Swq1q/zoegpQ7dcrc=";
+    hash = "sha256-FvYC4UioPij8t48CkFxuxJedOOPOf7Gw8cFj99Eyfbo=";
+  } // {
+    # To help nix-update generating correct Diff url:
+    rev = "refs/tags/v${version}";
   };
```

Comment:
This looks too hacky. Does `gitUpdater` with `url` and `rev-prefix` work?

Reply:
just tried, and I'm getting
```
+ echo '   git commit -m '\''simplex-chat-desktop: None -> 5.5.0-beta.2'\'''
+ /nix/store/92nw1lkfpm3rxcsaw625zvbddq74cg07-common-updater-scripts/bin/update-source-version simplex-chat-desktop 5.5.0-beta.2
update-source-version: error: Couldn't figure out where out where to patch in new version in 'simplex-chat-desktop'!


--- SHOWING ERROR LOG FOR simplex-chat-desktop-5.4.2 ----------------------
The update script for simplex-chat-desktop-5.4.2 failed with exit code 1
```
maybe I can just remove the rev and open an issue on nix-update to add on option to skip generating the Diff url.

Reply:
ah, that's because I forgot to cherry-pick https://github.com/NixOS/nixpkgs/pull/280492 before running the script.
Now it works. (but #280492 need to be merged first for update script to work)